src/main/java/Domain/Game/Season.java

```diff
@@ -97,12 +125,49 @@ public static boolean isGoodYearsFormat(String years) {
     }
 
 
-    private boolean checkTeamsEquals(List<Team> teams1,List<Team> teams2) {
-        for (Team team : teams1){
-            if(!teams2.contains(team)){
+    private boolean checkTeamsEquals(List<Team> teams1, List<Team> teams2) {
+        for (Team team : teams1) {
+            if (!teams2.contains(team)) {
                 return false;
             }
         }
-        return  true;
+        return true;
+    }
+
+    /**
+     * Checks if a given referee role is assigned to this season
+     *
+     * @param refereeRole - Referee - the referee to verify if he is assigned to this season
+     * @return - boolean - true if th referee is assigned to this season.
+     */
+    public boolean doesContainsReferee(Referee refereeRole) {
+        if (refereeRole != null) {
+            if (this.referees.contains(refereeRole)) {
+                return true;
+            }
```

Comment:
אני חושב שכדאי לעשות בדיקה יותר פנימית, כלומר לבדוק מזהה ייחודי כמו 
username
ברשימת השופטים ובשופט שהתקבל כפרמטר, מאשר לעשות בדיקה
by refernce.
(Unless you already overrided equals() at Role or something. **_If not, I highly recommend to override equals() at Referee_**, like we did at Coach)

Reply:
פה לא הייתי עושה בדיקה מעמיקה יותר.
השוואה עושים עם equals.
אני אדרוס אותו בRFEREE.